templates/feature_group/sdk_create_post_request.go.tpl

```diff
@@ -0,0 +1,10 @@
+
+// If creating, requeue with wait until status becomes created.
+if err == nil {
+        if foundResource, err := rm.sdkFind(ctx, desired); err != ackerr.NotFound {
```

Comment:
look at how other resources handle this. just need to set appropriate conditions post create